state/application.go

```diff
@@ -2190,12 +2199,13 @@ func (a *Application) GetScale() int {
 
 // ChangeScale alters the existing scale by the provided change amount, returning the new amount.
 // This is used on CAAS models.
-func (a *Application) ChangeScale(scaleChange int) (int, error) {
+func (a *Application) ChangeScale(scaleChange int, attachStorage []names.StorageTag) (int, error) {
```

Comment:
Hi @wallyworld 

Can I get your opinion on the change of this function? It's the core logic we make sure the ChangeScale will have the same behavior when deploy the application with --attach-storage.